src/scripts/create_schibsted.py

```diff
@@ -0,0 +1,110 @@
+"""Create the Schibsted summarisation dataset."""
+
+import fasttext
+import pandas as pd
+from constants import MAX_NUM_CHARS_IN_ARTICLE, MIN_NUM_CHARS_IN_ARTICLE
+from datasets import Dataset, DatasetDict, Split, load_dataset
+from huggingface_hub import HfApi, hf_hub_download
+from requests import HTTPError
+
+
+def main():
+    """Create the Schibsted summarisation dataset and upload to HF Hub."""
+    dataset_id = "Schibsted/schibsted-article-summaries"
+
+    dataset = load_dataset(dataset_id, token=True)
+    assert isinstance(dataset, DatasetDict)
+
+    # Rename validation split to val
+    dataset["val"] = dataset.pop("validation")
+
+    # Rename article_text_all to text
+    dataset = dataset.rename_columns(
+        column_mapping=dict(article_text_all="text", summary="target_text")
+    )
+
+    # Add article title to text, separated by a colon
+    dataset = dataset.map(lambda x: {"text": f"{x['article_title']}: {x['text']}"})
+
+    # Summary is a list of strings, but should be a single string
+    dataset = dataset.map(lambda x: {"target_text": " ".join(x["target_text"])})
```

Comment:
Are we sure that the list is not a list of different summaries? Just to make sure we don't end up with 3x summaries as the target text.

Reply:
I went through multiple samples, and it is definitely my impression that it is not a list of summaries, but one coherent summary.

Some examples:

```
['Friidrettsmiljøet kritiserer den planlagte turnering «Enhanced Games» i 2025 der doping er tillatt.', 'Turneringen skal utforske menneskekroppens ekstreme grenser. Arrangørene omtales som «fremtidens OL».', 'Internasjonalt friidrettsforbund og Wada advarer mot konseptet.', 'Over 900 utøvere har vist interesse for «Enhanced Games», blant annet tidligere svømmeverdensmester James Magnussen.', 'Både doped og ikke-dopede utøvere får delta side om side uten krav om doping.']
```

```
['Ekstremistgruppa Den islamske staten (IS) hevder å stå bak terrorangrepet i konserthallen Crocus i Moskva fredag kveld i Russland', 'Russiske myndigheter har antydet at angrepet har tilknytninger til Ukraina, og har sagt at de ble pågrepet nær den ukrainske grensen. ', 'Sikkerhetseksperter mener det er nærliggende og tro at IS snakker sant når de sier de står bak. ']
```

```
['Bokselegenden Mike Tyson (58) gjør endringer i trening for kampen mot YouTuber Jake Paul (27)', 'Kampen finner sted 20. juli på AT&T Stadium i Texas', 'Tyson legger ned podkasten "Hotboxin\' with Mike Tyson" for å fokusere på trening og kamp', 'Mange kritikere, inkludert boksekommentator Thomas Hansvoll, ser på kampen som et "freakshow"', 'Usanne rykter om beskyttende hodeplagg og tyngre hansker avkreftet av begge bokserne']
```

Reply:
Ah yeah I see, they seem to be referring to each other and not having any content overlap, so sounds good!